content/en/blog/_posts/2021-07-12-Kubernetes-Release-Cadence/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: blog
+title: "Kuberentes Release Cadence Change: Here’s What You Need To Know"
+date: 2021-07-01
+slug: new-kubernetes-release-cadence
+---
+
+REF: https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/
+
+
+**Authors**: Celeste Horgan, Adolfo García Veytia, James Laverack, Jeremy Rickard
+
+On April 23, 2021, the Release Team merged a Kubernetes Enhancement Proposal (KEP) changing the Kubernetes release cycle from four releases a year (once a quarter) to three releases a year. 
+
+This blog post is a high level overview post for what this means for the Kubernetes community's contributors and maintainers.
+
+## What's changing and when
+
+Starting with the [Kubernetes 1.22 release](https://www.kubernetes.dev/resources/release/), a lightweight policy will drive the creation of each release schedule. This policy states:
+
+* The first Kubernetes release of a calendar year should start at the second or third
+   week of January to provide people more room after coming back from the
+   winter holidays.
+* The last Kubernetes release of a calendar year should be finished by the middle of
+   December.
+* A Kubernetes release cycle has a length of approximately 15 weeks.
+* The week of KubeCon + CloudNativeCon is not considered a 'working week' for SIG Release. The Release Team will not hold meetings or make decisions in this period.
+* An explicit SIG Release break of at least two weeks between each cycle will
+   be enforced.
+
+As a result, Kubernetes will follow a three release pear year cadence. Kubernetes 1.23 will be the final release of the 2021 calendar year. This new schedule creation policy results in a very predictible release schedule, allowing us to forecast upcoming release dates:
+
+
+*Proposed Kubernetes Release Schedule for the remainder of 2021*
+
+| Week Number in Year | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 35 | 3 | 1 (August 23) | | 
+| 50 | 3 | 16 (December 07) | KubeCon + CloudNativeCon NA Break (Oct 11-15) | 
+
+*Proposed Kubernetes Release Schedule for 2022*
+
+| Week Number in Year | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 1  | 1 | 1 (January 03) | |
+| 15 | 1 | 15 (April 12) | | 
+| 17 | 2 | 1 (April 26) | KubeCon + CloudNativeCon EU likely to occur |
+| 32 | 2 | 15 (August 09) | |
+| 34 | 3 | 1 (August 22 | KubeCon + CloudNativeCon NA likely to occur |
+| 49 | 3 | 14 (December 06) |
+
+These proposed dates reflect only the start and end dates, and they are subject to change. The Release Team will select dates for enhancement freeze, code freeze, and other milestones at the start of each release. Feedback from prior releases will feed into this process.
+
+## What this means for end users 
```

Comment:
You don't mention any downside for changing the release cadence. If there is none, then that's fine. But I'm wondering if there will be some impact of fixes or enhancements having to wait for a longer period of time. Is there something built into the new cadence that can mitigate those potential negative effects?

Reply:
I can add this, I think we addressed it in the KEP so I'll paraphrase that content here, if that works?

Reply:
Yes. That was the only thing I wondered after reading the whole document.

Reply:
The negatives really come down to slower feature graduation and velocity. There isn't much that can be mitigated, it's just a result of the current policy around graduation (number of releases) and time. I've added information for users to be aware of the impact.